Backpack-SwiftUI/SearchInputSummary/Classes/BPKSearchInputSummary.swift

```diff
@@ -35,22 +36,33 @@ public struct BPKSearchInputSummary: View {
     @Binding private var text: String
     @FocusState private var focused: Bool
     private let placeholder: String
-    private let inputPrefix: InputPrefix
-    private var state: State = .default
+    private let inputPrefix: InputPrefix?
+    private var state: Style = .default
```

Comment:
```suggestion
    private var style: Style = .default
```
Not an issue, just to be consistent with new naming/approach.